diff --git a/logic/slides.ts b/logic/slides.ts
index cb156a18af745ff4c2028bc6515683ec2e45c277..482111a3fc0eaef93791942200513cfaac9de706 100644
--- a/logic/slides.ts
+++ b/logic/slides.ts
@@ -21,7 +21,7 @@ export function getSlidePath(
     route = getSlide(route)!
   const no = route.meta.slide?.frontmatter.routeAlias ?? route.no
   const path = exporting ? `export/${no}` : presenter ? `presenter/${no}` : `${no}`
-  return `${import.meta.env.BASE_URL}${path}`
+  return `/${path}`
 }
 
 export function useIsSlideActive() {
